README_cmake.md

```diff
@@ -67,12 +67,11 @@ we strongly suggest reading through the [CMake documentation][cmake-docs] first.
 
 ## Installing CMake
 
-Halide requires at least version 3.22, which was released in November 2021.
-Fortunately, getting a recent version of CMake couldn't be easier, and there are
-multiple good options on any system to do so. Generally, one should always have
-the most recent version of CMake installed system-wide. CMake is committed to
-backwards compatibility and even the most recent release can build projects over
-a decade old.
+Halide requires at least version 3.24. Fortunately, getting a recent version of
+CMake couldn't be easier, and there are multiple good options on any system to do
```

Comment:
I'd also suggest listing a few of the options, because it was non-obvious to me that you could acquire one via pip.

How about (e.g. homebrew, pip, snap)

Reply:
Oh never mind I totally missed that this is spelled out in detail below.

Reply:
We should put something like "For guidance on how to acquire a new enough version of cmake, see (link to README_cmake.md)." in the main README

Reply:
Done :)

Reply:
> We should put something like "For guidance on how to acquire a new enough version of cmake, see (link to README_cmake.md)." in the main README

This is a total threadjack, but IMHO it's long past time to consolidate our READMEs into a single file with multiple sections -- a prospective user shouldn't have to figure out which README to read to find what they need, they should just be able to open the one-and-only README and look at the TOC and/or search for stuff.